src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public abstract class Truck extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("Truck started its work");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Truck stop working");

```

Comment:
The stop message 'Truck stop working' has a grammatical mistake and could be more informative. Consider using 'Truck has stopped working' and specify the type of truck if possible.